### PR TITLE
add DEV.md, quick fixes and minor updates

### PR DESCRIPTION
**Pull request by wonderingabout**

### CODE REVIEW

File: connection.js [hunk 1]
```diff
@@ -691,8 +691,8 @@ function getArgNameStringsGRU(familyNameString) {
 
 function rankToString(r) {
     const R = Math.floor(r);
-    if (R >= 30)  return `${R - 30 + 1}d`; // R >= 30: 1 dan or stronger
```
Comment (Dorus) [inline connection.js:694]: This was clearer with the `:` in i think.

**Comment by Dorus** [comment]
This one is ok to merge after my 1 comment.

File: connection.js (hunk 1 above)
Comment (wonderingabout) [reply connection.js:694]: ok, i wanted to remove as much punctuation as possible like markdown lint does, but it is indeed clearer in that case :1st_place_medal:

Comment (wonderingabout) [reply connection.js]: fixed

**Pull request merged**